### PR TITLE
[zh]Update content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md [hunk 1]
```diff
@@ -1409,13 +1409,27 @@ The following is an example procedure to upgrade from `v1beta1` to `v1`.
...
-    `storedVersions` field.
+3.  Remove `v1beta1` from the CustomResourceDefinition `status.storedVersions` field.
 -->
 1.  在 CustomResourceDefinition 文件中将 `v1` 设置为存储版本，并使用 kubectl 应用它。
     `storedVersions`现在是`v1beta1, v1`。
 2.  编写升级过程以列出所有现有对象并使用相同内容将其写回存储。
     这会强制后端使用当前存储版本（即 `v1`）写入对象。
-3.  通过从 `storedVersions` 字段中删除 `v1beta1` 来更新 CustomResourceDefinition
-    的`Status`。 
+3.  从 CustomResourceDefinition 中删除 `v1beta1` 的 `status.storedVersions` 字段。
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md:1418]:
```suggestion
3.  从 CustomResourceDefinition  的 `status.storedVersions` 字段中删除 `v1beta1`。
```

File: content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md [hunk 2]
```diff
@@ -1409,13 +1409,27 @@ The following is an example procedure to upgrade from `v1beta1` to `v1`.
...
 
+<!--
+The `kubectl` tool currently cannot be used to edit or patch the `status` subresource on a CRD: see the [Kubectl Subresource Support KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/2590-kubectl-subresource) for more details.
+
+The easier way to patch the status subresource from the CLI is directly interacting with the API server using the `curl` tool, in this example:
+-->
+{{< note >}}
+`kubectl` 工具目前不能用于编辑或修补 CRD 上的 `status` 子资源:请参阅 [kubectl子资源支持KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/2590-kubectl-subresource) 了解更多细节。
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md:1426]:
```suggestion
`kubectl` 工具目前不能用于编辑或修补 CRD 上的 `status` 子资源：请参阅
[kubectl Subresource Support KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/2590-kubectl-subresource)
了解更多细节。
```

File: content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md [hunk 3]
```diff
@@ -1409,13 +1409,27 @@ The following is an example procedure to upgrade from `v1beta1` to `v1`.
...
+
+从CLI给 `status` 子资源打补丁的更简单的方法是使用 `curl` 工具直接与API服务器交互，示例:
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md:1428]:
```suggestion
从 CLI 给 `status` 子资源打补丁的更简单的方法是使用 `curl` 工具直接与 API 服务器交互，示例:
```

**Comment by mengjiao-liu** [comment]
The code looks good to me, but there are too many commits, so please follow the requirements below for    `Squash `.
See [Squash commits](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#squash-commits) for more details.

**Comment by kinzhi** [comment]
Please help to review this PR

**Comment by mengjiao-liu** [comment]
/approve
/lgtm
Thanks!

**Pull request merged**